src/cmd_line/subparsers/sort.ts

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+import * as node from "../commands/sort";
+
+export function parseSortCommandArgs(args : string) : node.SortCommand {
+
+  let reverse_ = false;
```

Comment:
Why the trailing underscore? Also you could write `const reverse = args !== null && args.indexOf("!") >= 0;`